config_kz_2011_discount10_coalexit.yaml

```diff
@@ -17,9 +18,8 @@ load_options:
 
 electricity:
   extendable_carriers:
-    Generator: [solar, onwind, offwind-ac, offwind-dc]
+    Generator: [solar, onwind, offwind-ac, offwind-dc, coal]
     Store: [battery, H2]
-  conventional_carriers: [nuclear, oil, OCGT, CCGT, lignite, geothermal, biomass]
```

Comment:
If yes I'm happy to merge

Reply:
Yes, it works fine, because for coal-exit we need to have all conventional_carriers including `coal`. So the carriers from default config fits perfectly.